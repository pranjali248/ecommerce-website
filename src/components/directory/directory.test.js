import React from "react";
import { shallow } from "enzyme";
import Directory from "./directory.component";

describe("Testing of directory component", () => {
  it("Shallow snapshot testing of directory component", () => {
    expect(shallow(<Directory />)).toMatchSnapshot();
  });
});
