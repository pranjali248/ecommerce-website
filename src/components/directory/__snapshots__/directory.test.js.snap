// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing of directory component Shallow snapshot testing of directory component 1`] = `ShallowWrapper {}`;
